bin.js

```diff
@@ -271,11 +271,9 @@ function build (entry, outputDir, argv, done) {
     if (argv.uglify !== false && argv.uglify !== 'false') {
       log.debug('uglify starting')
       js = uglify.minify(js, {
-        fromString: true,
-        compress: true,
-        mangle: true,
-        filename: file,
-        sourceMaps: false
+        sourceMap: {
+          filename: file
```

Comment:
Unless I missed something, I am not sure if there is another way to pass in the filename except through `sourceMap`, which is why I kept it there. 

that `cat`, are those the opts you're thinking of keeping enabled?